TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementCheckers.cs

```diff
@@ -0,0 +1,27 @@
+﻿using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Compiler.Parser.Generated;
+using TypeCobol.Compiler.Sql.CodeElements.Statements;
+
+namespace TypeCobol.Compiler.Sql.CodeElements
+{
+    class SqlCodeElementCheckers
+    {
+        public static void OnSetAssignmentStatementChecker(CodeElement codeElement,
+            CodeElementsParser.SetAssignmentStatementContext context)
+        {
+            var statement = (SetAssignmentStatement) codeElement;
+            foreach (var assignment in statement.Assignments)
+            {
+                if (assignment.Values != null && assignment.Targets != null &&
```

Comment:
Ok either we ensure that both list are always non-null or we consider a null list is equivalent to an empty list.

Reply:
I didn't understand the second one .
You mean i can verify si les listes sont vides ``` !assignment.Values.Any() && !assignment.Values.Any()``` ?